compiler/syntax/src/res_ast_debugger.ml

```diff
@@ -707,9 +707,47 @@ module SexpAst = struct
           ]
       | Pexp_extension ext ->
         Sexp.list [Sexp.atom "Pexp_extension"; extension ext]
+      | Pexp_jsx_fragment (_, children, _) ->
```

Comment:
@cristianoc what should all this code be?
I'm not sure what the lore is here.
Any pointers?

Reply:
Not really sure if debugger is even used anymore. @zth any ideas?

Reply:
I never use it at least. Void the new nodes in there and see if anyone misses them?

Reply:
@cristianoc what do you mean with debugger here?

Reply:
Following the use of the module, it seems to be used in the print tests here for the `sexp` file:
```
    $DUNE_BIN_DIR/res_parser $resIntf -print sexp $file > $sexpAst1
    $DUNE_BIN_DIR/res_parser $resIntf -print res $file > $rescript1
```

which I believe are files only kept around when the roundtrip syntax tests fail.
Can't remember if they're actually useful, or the generated .res files are enough.

In any case, it's just something for debugging tests, so the details of how things are printed are not important.